pkg/util/util/util.go

```diff
@@ -45,9 +45,9 @@ func RandIDWithLen(idLen int) (id string, err error) {
 
 func GetAuthKey(token string, timestamp int64) (key string) {
 	token = token + fmt.Sprintf("%d", timestamp)
-	md5Ctx := md5.New()
-	md5Ctx.Write([]byte(token))
-	data := md5Ctx.Sum(nil)
+	sha256Ctx := sha256.New()
```

Comment:
what's the timestamp here?what's the reason you said that old client won't work? becaue if the timestamp is current timestamp, then to me this function is probably just called by one party to generate a key, not by two parties to generate the same key. But I'm just guessing.

Reply:
@jzhoucliqr Client is not able to authenticate with server with these changes. Need to deep dive into the code to find out how it is being used, but it feels like we will need to host 2 proxy server for a while.

```
(base) rishi@Rishis-MacBook-Pro ~ $ k logs -f spectro-proxy-7f875568d8-gr7c7
Defaulted container "frpc" out of: frpc, frpc-init (init)
2023/02/06 01:19:33 [E] [service.go:340] token in login doesn't match token from configuration
2023/02/06 01:19:33 [W] [service.go:128] login to server failed: token in login doesn't match token from configuration
token in login doesn't match token from configuration
(base) rishi@Rishis-MacBook-Pro ~ $
```